src/modules/beamos/start_chroot_script

```diff
@@ -483,6 +483,7 @@ pushd /home/pi
   else
     # Remove all of the branch: some-branch keys in the config.yaml file
     yq e 'del(.plugins.softwareupdate.checks.*.branch)' -i .octoprint/config.yaml
+    yq e 'del(.plugins.softwareupdate.checks.*.current)' -i .octoprint/config.yaml
```

Comment:
?

Reply:
This is to delete the current hash, as we don't need them anymore (version based update)

Reply:
But we can keep them for develop variants